src/views/viewer/HomeView.tsx

```diff
@@ -13,7 +13,13 @@ export const HomeView: React.FC = () => {
   const channelIdIn = followedChannels.map((channel) => channel.id)
   const anyFollowedChannels = channelIdIn.length > 0
 
-  const { videosConnection, loading, error } = useVideosConnection(
+  const { mostViewedVideos, loading: mostViewedVideosLoading, error: mostViewedVideosError } = useMostViewedVideosIds({
+    limit: 200,
+    period: 30,
```

Comment:
what is period?

Reply:
It's number of days from today.

Reply:
Might be worth looking for a better name here I think, something on the lines of `uploadedBeforeDays`

Reply:
Hmm don't you think it would be better to keep it consistent with Orion query? Btw it's not days of upload but sum of view event timestamps.

Reply:
keeping it consistent with Orion is a good idea but I do think it's quite unclear from just looking at it as first thought would be it was days from upload

Reply:
Ok, I buy this 😉

Reply:
Yeah, maybe we should have used better name in Orion